transaction/std/src/transaction_builder.rs

```diff
@@ -22,12 +22,22 @@ use mc_transaction_core::{
 };
 use mc_util_from_random::FromRandom;
 use rand_core::{CryptoRng, RngCore};
+use std::cmp::Ordering;
 
-/// This is a datatype for the comparer of the transaction outputs
-pub type TxComparerFn = fn(&TxOut, &TxOut) -> core::cmp::Ordering;
+/// A trait used to compare the transaction outputs
+pub trait TxOutputsOrdering {
+    /// comparer method
+    fn cmp(&self, a: &CompressedRistrettoPublic, b: &CompressedRistrettoPublic) -> Ordering;
```

Comment:
This probably doesn't need self, it can be a static method.

Reply:
fixed